beacon-chain/utils/spectest/shuffle_yaml_test.go

```diff
@@ -0,0 +1,84 @@
+package spectest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+	"reflect"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/go-yaml/yaml"
+	"github.com/prysmaticlabs/prysm/beacon-chain/utils"
+	"github.com/prysmaticlabs/prysm/shared/params/spectest"
+)
+
+func TestShuffleYaml(t *testing.T) {
+	yamlDir := "./"
+	ext := ".yaml"
+	files, err := ioutil.ReadDir(yamlDir)
+	if err != nil {
+		t.Fatalf("could not read YAML tests directory: %v", err)
+	}
+
+	for _, file := range files {
+		if file.IsDir() {
+			continue
+		}
+		extension := string(file.Name()[len(file.Name())-5:])
+		if err != nil || extension != ext {
+			continue
+		}
+		filePath := path.Join(yamlDir, file.Name())
+		// #nosec G304
+		data, err := ioutil.ReadFile(filePath)
+		if err != nil {
+			t.Fatalf("could not read YAML tests directory: %v", err)
+		}
+		shuffleTest := &ShuffleTest{}
+		if err := yaml.Unmarshal(data, shuffleTest); err != nil {
+			t.Fatalf("could not unmarshal YAML file into test struct: %v", err)
+		}
+		if shuffleTest.Config == "minimal" {
```

Comment:
This if condition is redundant
```suggestion
		if err := spectest.SetConfig(shuffleTest.Config); err != nil {
				t.Fatal(err)
			}
```

Reply:
thanks

looks much better